tests/unit/src/test/scala/tests/DebugDiscoverySuite.scala

```diff
@@ -91,6 +92,32 @@ class DebugDiscoverySuite
     } yield assertNoDiff(output, "oranges are nice")
   }
 
+  test("run-scala-cli-script") {
```

Comment:
Could you also add a test for breakpoints? Similar to how it's done in `BaseBreakpointDapSuite` though we might want to add it somewhere else since that suite is used for Bloop and sbt